mobile/lib/modules/asset_viewer/ui/top_control_app_bar.dart

```diff
@@ -39,7 +39,9 @@ class TopControlAppBar extends HookConsumerWidget {
     const double iconSize = 22.0;
     final a = ref.watch(assetWatcher(asset)).value ?? asset;
     final album = ref.watch(currentAlbumProvider);
-    final comments = album != null
+    final comments = album != null &&

```

Comment:
Ah, good catch. We don't need to check if the asset's remoteId is null though. So the following should be enough

```dart
final comments = album?.remoteId != null ? <provider> : 0
```